app/controllers/confirmations_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ConfirmationsController < Devise::ConfirmationsController
+  FLASH_MESSAGE = "Email sent! Please contact support at %<email>s if you are "\
+    "having trouble receiving your confirmation instructions.".freeze
+
+  def create
+    self.resource = resource_class.send_confirmation_instructions(resource_params)
+    resource.errors.clear # Don't leak user information, like paranoid mode.
```

Comment:
It seems by overriding the controller we lost paranoid mode, at least I could see error messages. Instead of digging deep into Devise internals, I decided to just clear the resource errors manually.